barretenberg/cpp/src/barretenberg/plonk/composer/composer_lib.hpp

```diff
@@ -48,4 +48,79 @@ std::shared_ptr<plonk::verification_key> compute_verification_key_common(
     // silencing for now but need to figure out where to extract type of VerifierCrs from :-/
     std::shared_ptr<bb::srs::factories::VerifierCrs<curve::BN254>> const& vrs);
 
+/**
```

Comment:
This is now used only by plonk so I moved it to this plonk only composer lib so it can be deleted with everything else.